Sources/BugsnagPerformance/Public/BugsnagPerformanceSpanOptions.m

```diff
@@ -6,12 +6,14 @@
 //  Copyright © 2023 Bugsnag. All rights reserved.
 //
 
+#import <Foundation/Foundation.h>
+#import <BugsnagPerformance/BugsnagPerformanceSpan.h>
 #import <BugsnagPerformance/BugsnagPerformanceSpanOptions.h>
 
 @implementation BugsnagPerformanceSpanOptions
 
 + (instancetype)optionsWithStartTime:(NSDate *)startTime
-                       parentContext:(id<BugsnagPerformanceSpanContext>)parentContext
+                       parentContext:(BugsnagPerformanceSpan *)parentContext
```

Comment:
This is a public API change so we'll have to mark it as breaking in the CHANGELOG